ui/admin/app/routes/scopes/scope/targets/new.js

```diff
@@ -3,14 +3,24 @@ import Route from '@ember/routing/route';
 export default class ScopesScopeTargetsNewRoute extends Route {
   // =methods
 
+  // =attributes
+  queryParams = {
+    type: {
+      refreshModel: true,
+    },
+  };
   /**
    * Creates a new unsaved target in current scope.
    * @return {TargetModel}
    */
-  model() {
+
+  model(params) {
     const scopeModel = this.modelFor('scopes.scope');
+    if (!params.type) {
+      params.type = 'tcp'; //tcp is the default type
+    }
```

Comment:
Are we able to test this functionality?

Reply:
I might create a new local variable too, rather than assigning into the params object.  We receive this object from Ember internals.  Probably there's no harm but good to be safe.

Reply:
@cameronperera the acceptance test could attempt to navigate to the new route with no type, then check that the TCP form is displayed.

Reply:
@cameronperera the display test case is covered in [PR](https://github.com/hashicorp/boundary-ui/pull/1027) and we could discuss refactoring the CRUD test files offline tomorrow.

Reply:
discussed offline and @cameronperera will be adding a few more test cases to ssh target CRUD work in a separate PR.